django-backend/fecfiler/contacts/views.py

```diff
@@ -32,11 +33,24 @@ class ContactViewSet(CommitteeOwnedViewSet):
     The queryset will be further limmited by the user's committee
     in CommitteeOwnedViewSet's implementation of get_queryset()
     """
+
     queryset = (
-        Contact.objects.annotate(transaction_count=Count("transaction"))
+        Contact.objects
+        .annotate(transaction_count=Count("transaction"))
+        .annotate(sort_name=Concat("name", "last_name", Value(" "), "first_name", output_field=CharField()))
```

Comment:
if you use alias for sort_name, you won't need to remove it in the serializer because it will only exist during the query

Reply:
Good catch